modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/Tooltip.kt

```diff
@@ -0,0 +1,233 @@
+@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
```

Comment:
No need to change it but I hope this doesn't come back to us at some point if they decide to change internals of the Material library.